File: Samples/android/Java/app-v7-java/src/main/java/com/vungle/samples/samplejava/MainActivity.java
package com.vungle.samples.samplejava;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.vungle.samples.samplejava.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    private ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
    }
}

File: Samples/android/Android-SDK-Demo-App/app-v7-java/src/main/java/com/vungle/samples/samplejava/MainActivity.kt
package com.vungle.samples.samplejava

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.vungle.samples.samplejava.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private var binding: ActivityMainBinding? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(
            layoutInflater
        )
        setContentView(binding!!.root)
    }
}